src/main/java/bio/terra/service/snapshot/flight/export/SnapshotExportCreateParquetFilesStep.java

```diff
@@ -37,13 +37,18 @@ public SnapshotExportCreateParquetFilesStep(
 
   @Override
   public StepResult doStep(FlightContext context) throws InterruptedException, RetryException {
+    String flightId = context.getFlightId();
     FlightMap workingMap = context.getWorkingMap();
     Snapshot snapshot = snapshotService.retrieve(snapshotId);
     GoogleBucketResource exportBucket =
         workingMap.get(SnapshotWorkingMapKeys.SNAPSHOT_EXPORT_BUCKET, GoogleBucketResource.class);
 
-    List<String> paths =
-        bigQueryPdao.exportTableToParquet(snapshot, exportBucket, context.getFlightId());
+    final List<String> paths;
+    if (workingMap.containsKey(SnapshotWorkingMapKeys.SNAPSHOT_EXPORT_GSPATHS_FILENAME)) {
```

Comment:
This will change once we have the API parameter

Reply:
When would we hit the else condition here? Isn't SNAPSHOT_EXPORT_GSPATHS_FILENAME always set in SnapshotExportDumpFirestoreStep and that step always runs?

Reply:
Ah, the SnapshotExportDumpFirestoreStep will be moved behind the api flag according to the comment in the export flight, right?

Reply:
Yup!

Reply:
So, this code won't change with follow-on PR. The SnapshotExportDumpFirestoreStep will just optionally run.